core/src/test/java/org/elasticsearch/search/aggregations/bucket/IPv4RangeTests.java

```diff
@@ -869,4 +870,77 @@ public void emptyAggregation() throws Exception {
         assertThat(buckets.get(0).getToAsString(), equalTo("10.0.0.10"));
         assertThat(buckets.get(0).getDocCount(), equalTo(0l));
     }
+
+    @Test
+    public void mask0() {
+        SearchResponse response = client().prepareSearch("idx")
+                .addAggregation(ipRange("range")
+                        .field("ip")
+                        .addMaskRange("0.0.0.0/0"))
+                .execute().actionGet();
+
+        assertSearchResponse(response);
+
+        Range range = response.getAggregations().get("range");
+        assertThat(range, notNullValue());
+        assertThat(range.getName(), equalTo("range"));
+        List<? extends Bucket> buckets = range.getBuckets();
+        assertThat(range.getBuckets().size(), equalTo(1));
+
+        Range.Bucket bucket = buckets.get(0);
+        assertThat((String) bucket.getKey(), equalTo("0.0.0.0/0"));
+        assertThat(bucket.getFromAsString(), nullValue());
+        assertThat(bucket.getToAsString(), nullValue());
+        assertThat(((Number) bucket.getTo()).doubleValue(), equalTo(Double.POSITIVE_INFINITY));
+        assertEquals(255l, bucket.getDocCount());
+    }
+
+
+    @Test
+    public void mask0SpecialIps() throws Exception {
```

Comment:
@jpountz This creates and index and adds documents to it. Should the first part be moved to the `setupSuiteScopeCluster(*)`?